### ISSUE TITLE
Looking for maintainers

### ISSUE THREAD

**Issue by baopham** [baopham/laravel-dynamodb#213]
I've been away for quite some time due to not having enough bandwidth. I'm not sure what my bandwidth will be like for the next few months so would love to get some help out there. 

Let me know if you're interested. I'm hoping to find someone who would be super familiar with PHP, Laravel, DynamoDB and have passion for clean code (with tests :) ).

**Comment by rennokki** [comment]
I have forked your repo on my profile a while ago. I can take time to work with it.

**Comment by JackPriceBurns** [comment]
I would be willing to help maintain this package, I will be using it actively in a future project of mine and already have a few improvements I'd like to make.

**Comment by baopham** [comment]
Awesome! Let me leave this open for a few days and then we finalize. I also need to sort out Travis / Scrutinizer accounts and create a channel (probably Slack) for us to communicate.

**Comment by baopham** [comment]
I just added you two to the collaborators list. Let's try to come up with a way to collaborate (e.g. when to accept PR, when to make a new release, channel to discuss things)?

**Comment by JackPriceBurns** [comment]
Sounds good, do we want to make a slack?

**Comment by baopham** [comment]
Yep, here it is: https://laravel-dynamodb.slack.com

**Comment by JackPriceBurns** [comment]
Can you send me an invite to jackpriceburns@outlook.com please 🙂

**Comment by baopham** [comment]
done!

**Comment by thomasedwards** [comment]
I’m more than happy to help out triage issues, this repo is very important to me!

**Comment by baopham** [comment]
Thanks @thomasedwards ! I added you to the repo. Feel free to join our Slack channel too :)

**Comment by katakutu** [comment]
can add me to this repo thanks

**Comment by baopham** [comment]
@katakutu done. Send me an email if you want an invite to our Slack channel :)

**Comment by AidanLaycock** [comment]
Hi, could I be invited to the slack as I'd like to get more involved! - aidanl_@hotmail.co.uk

**Comment by xengulai** [comment]
Can I get invited to the Slack? I'm working on a feature and have some questions. jaysen [at] jaysen [dot] net

**Comment by JackPriceBurns** [comment]
Hey @xengulai, I've send an invite!

**Comment by thebatclaudio** [comment]
Hello, can I get invited to Slack? labarbera.claudio@gmail.com

**Comment by thebatclaudio** [comment]
Hello to everyone in this thread, someone can review and accept this PR https://github.com/baopham/laravel-dynamodb/pull/254 ?
I've been waiting for over a month. Is this repository maintained or is it better to fork and publish it again on packagist? Thank you

**Comment by nelson6e65** [comment]
Is this project abandoned?

**Comment by baopham** [comment]
yeah I'm not sure if the previous maintainers can still commit to supporting this project. If anyone is interested, let me know.

**Comment by nelson6e65** [comment]
I'm interested. I'm actively using it for the moment. I would like to add some improvements, not sure about to be the main mainteiner. Anyway, I would like to improve support for active versions of Laravel.

I have one project using Laravel 5.8, but there are plans to upgrade it to a recent version.

With a security support policy, we should only support versions already being supported by Laravel team and maybe a popular version (like me, using 5.8 😅).

I can help in long term with:
- GitHub Actions (initially by moving the current TravisCI),
- Improve code style (PHPCS)
- Improve tests, PHPUnit (and maybe moving to PestPHP gradually 🤔)
- Improve codebase with new PHP versions flavours, as possible. 
- Some minor revisions of features/fixes until I get some experience in this cosebase.

**Comment by baopham** [comment]
> I'm interested. I'm actively using it for the moment. I would like to add some improvements, not sure about to be the main mainteiner. Anyway, I would like to improve support for active versions of Laravel.
> 
> I have one project using Laravel 5.8, but there are plans to upgrade it to a recent version.
> 
> With a security support policy, we should only support versions already being supported by Laravel team and maybe a popular version (like me, using 5.8 😅).
> 
> I can help in long term with:
> 
> * GitHub Actions (initially by moving the current TravisCI),
> * Improve code style (PHPCS)
> * Improve tests, PHPUnit (and maybe moving to PestPHP gradually 🤔)
> * Improve codebase with new PHP versions flavours, as possible.
> * Some minor revisions of features/fixes until I get some experience in this cosebase.

awesome! I added you as a contributor. Love the plan 🙂

**Comment by katakutu** [comment]
iam interesting to this project. Maybe i can help nelson

**Comment by zoe-edwards** [comment]
I’m still about – I’ve been answering/closing comments but I don’t know too much about the internals to be comfortable approving PRs etc.

**Comment by nelson6e65** [comment]
Hello, @baopham @zoe-edwards . I was preparing my local environment, but I got some troubles with the local DynamoDB database. 😅 

Can I have some help on this? I'll have this Saturday to try again. (Should we enable [Discussions](https://docs.github.com/en/discussions)? 👀)

**Comment by zoe-edwards** [comment]
It’s not really something I’ve ever tried to do @nelson6e65, I’d recommend just using AWS for this as it’s quite inexpensive and more reliable.

**Comment by nelson6e65** [comment]
It is not necessary to use live version. Local version is enough. Actually, as it will be integrated into local and CI tests, it should not rely on any actual account having DynamoDB enabled. 😅

I installed the Docker version and seems to work, but my problem was about how to use the initialized database for testing. Maybe I'm missing a parameter to set up the database to use (as used with the bin located at /dynamodb_local directory). 🤔

**Comment by thebatclaudio** [comment]
Hello people,
I would like to become a maintainer too!

@nelson6e65 I have some problems too: I can configure local DynamoDB, but when i run phpunit I receive an error:

```
PHP Fatal error:  During inheritance of ArrayAccess: Uncaught ErrorException: Return type of Illuminate\Http\Request::offsetExists($offset) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /home/claudio/PhpstormProjects/laravel-dynamodb-github/vendor/laravel/framework/src/Illuminate/Http/Request.php:637
```

Maybe we should fix also tests.

**Comment by nelson6e65** [comment]
> Hello people, I would like to become a maintainer too!
> 
> @nelson6e65 I have some problems too: I can configure local DynamoDB, but when i run phpunit I receive an error:
> 
> ```
> PHP Fatal error:  During inheritance of ArrayAccess: Uncaught ErrorException: Return type of Illuminate\Http\Request::offsetExists($offset) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /home/claudio/PhpstormProjects/laravel-dynamodb-github/vendor/laravel/framework/src/Illuminate/Http/Request.php:637
> ```

Yes, this library seems not be tested against PHP 8 yet. Feel free to create an issue about it.

> Maybe we should fix also tests (I never made tests, so I can be wrong)
> 
> EDIT: I read that you would like to implement PHPPest, if you want I can try to help you (and use this opportunity to learn how to make tests)

Of course! I'm also enabling GitHub Actions for running these tests. Just make sure to open an issue about the bug/feature before working hard in a PR.

**Comment by thebatclaudio** [comment]
Hey @baopham @nelson6e65,
can I become mantainer too? I'm using Laravel and DynamoDB often so I can help you to keep the project updated.
